Hestia/Hestia/DevicesScreen/AddDeviceScreen/PropertyCell.cs

```diff
@@ -0,0 +1,32 @@
+﻿using UIKit;
+using Foundation;
+using CoreGraphics;
+
+
+namespace Hestia.DevicesScreen.AddDeviceScreen
+{
+    public class PropertyCell : UITableViewCell
+    {
+        public UITextField inputField;
+        public PropertyCell(NSString cellId) : base(UITableViewCellStyle.Default, cellId)
+        {
+            // Create inputfield and add it to the cell
+            inputField = new UITextField();
+            ContentView.AddSubviews(new UIView[] {inputField});
+
```

Comment:
Try to keep formatting consistent and neat, no double whittling after functions, and no single at the end.